cmake/FindDPCPP.cmake

```diff
@@ -53,6 +53,7 @@ if(NOT "${DPCPP_SYCL_ARCH}" STREQUAL "")
   if("${DPCPP_SYCL_TARGET}" STREQUAL "nvptx64-nvidia-cuda")
     list(APPEND DPCPP_FLAGS "-Xsycl-target-backend")
     list(APPEND DPCPP_FLAGS "--cuda-gpu-arch=${DPCPP_SYCL_ARCH}")
+    list(APPEND DPCPP_FLAGS "-fgpu-inline-threshold=1000000;")
```

Comment:
If this still needed if we use `__attribute__((always_inline)) inline`?

Reply:
I remember that back in portFFT, we had to increase the inline threshold, in combination with always_inline,
hence I went ahead with this, and have not tested it individually. 
I can try without it, and will remove it if not required